util/network-devp2p/src/discovery.rs

```diff
@@ -275,7 +279,12 @@ impl<'a> Discovery<'a> {
 			if let Some(node) = node_to_ping {
 				self.try_ping(node, PingReason::Default);
 			};
-            TableUpdates{added, removed: HashSet::new()}
+
+            if !self.bootnodes.contains(&node_entry.id) {
+				Some(TableUpdates { added, removed: HashSet::new() })
```

Comment:
Instead we could just set `added` to be an empty `HashMap` I suppose. (i.e. the condition could be moved higher to just prevent adding to `added` collection.

Reply:
that would unnecessarily call `update_node` with an empty `TableUpdate`